pkg/obs/nodeserver.go

```diff
@@ -87,10 +88,15 @@ func (ns *nodeServer) NodePublishVolume(_ context.Context, req *csi.NodePublishV
 	if err := ns.Mount.MakeDir(targetPath); err != nil {
 		return nil, status.Errorf(codes.Internal, "Failed to make dir: %s, error: %v", targetPath, err)
 	}
-	if err := createCredentialFile(ns.Driver.cloud.Global.AccessKey, ns.Driver.cloud.Global.SecretKey); err != nil {
+
+	credentialFile := fmt.Sprintf("%s/%s", credentialDir, uuid.New().String())
+	defer deleteCredentialFile(credentialFile)
```

Comment:
this file should be deleted after creation, please move this line after lint 99

Reply:
done